app/views/application/_navigation.html.erb

```diff
@@ -13,24 +13,24 @@
         <li class="dropdown">
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Shop By Category <span class="caret"></span></a>
           <ul class="dropdown-menu">
-            <li><%= link_to "Category 1", items_path %></li>
-            <li role="separator" class="divider"></li>
-            <li><%= link_to "Category 2", items_path %></li>
-            <li role="separator" class="divider"></li>
-            <li><%= link_to "Category 3", items_path %></li>
+            <% Category.all.each do |category| %>
```

Comment:
We probably don't want to be hitting the database from the view--we can see if there's another way to do this, like somehow passing in an instance variable. Should we ask Sal and Ilana in PR?

Reply:
I think we can use content_for to do this, but it can't hurt to get their feedback